families/families.js

```diff
@@ -9,13 +9,46 @@ logoutButton.addEventListener('click', () => {
     logout();
 });
 
-function displayFamilies() {
-    // fetch families from supabase
-
-    // clear out the familiesEl
+async function displayFamilies() {
+    familiesEl.textContent = '';
+    const families = await getFamilies();
+    
 
     for (let family of families) {
-        // create three elements for each family, one for the whole family, one to hold the name, and one to hold the bunnies
+        const familyEl = document.createElement('div');
+        const nameEl = document.createElement('h3');
+        const bunniesEl = document.createElement('div');
+
+        bunniesEl.classList.add('bunnies');
+        familyEl.classList.add('family');
+
+        nameEl.textContent = family.name;
+
+        for (let bunny of family.fuzzy_bunnies) {
+            const bunnyEl = document.createElement('div');
+
+            bunnyEl.classList.add('bunny');
+            bunnyEl.textContent = bunny.name;
+
+            bunnyEl.addEventListener('click', async ()=> {
+                await deleteBunny(bunny.id);
+
+                const updateFamilies = await getFamilies();
+                displayFamilies(updateFamilies);
```

Comment:
Overall, this works well! Just pointing out one thing here on line 37 where you call display families with the argument updateFamilies. At the top of this function displayFamilies doesn't take an argument but I think it still works because you're writing to the back end and then displayFamilies makes that back end call and gets all those updated. Not causing any errors here but little things like that in the future can cause errors, just something to keep in mind.